docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,178 @@ up OpenAI authentication to run the code below.
             model_type="question-answering",
         )
 
+.. _llm-eval-model-endpoint:
+
+Evaluating with an MLflow Deployments Endpoint
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For MLflow >= 2.11.0, :py:func:`mlflow.evaluate()` supports evaluating a model endpoint by directly passing the MLflow Deployments endpoint URI to the ``model`` argument.
+This is particularly useful when you want to evaluate a deployed model hosted by a local `MLflow Deployments Server <../deployments/index.html>`_ or `Databricks Foundation Models API and External Models <https://docs.databricks.com/en/machine-learning/model-serving/score-foundation-models.html>`_, without implementing custom prediction logic to wrap it as an MLflow model or a python function.
```

Comment:
@B-Step62 the external models docs are separate: https://docs.databricks.com/en/generative-ai/external-models/index.html. Can we link those separately?

Reply:
Yeah but it appears the linked doc is designed for both
```
Databricks Model Serving supports Foundation Models API and external models for accessing foundation models and uses a unified OpenAI-compatible API and SDK for querying them.
```